main/guides/coreeval/README.md

```diff
@@ -0,0 +1,12 @@
+# Injecting Code to Agoric Testnet
+
+Agoric facilitates safely injecting code to the Agoric testnet, usually for the purpose of running a contract and modifying it based on the result of a governance vote. To do so, we submit a proposal using cosmos-sdk level API ([swingset.CoreEval](https://community.agoric.com/t/blder-dao-governance-using-arbitrary-code-injection-swingset-coreeval/99)). The proposal runs a script that runs the contract, and possibly runs an additional script, depending on the result of a governance vote.
```

Comment:
I took another stab at clarifying the last sentence, please consider with a grain of salt:

```suggestion
Agoric facilitates safely injecting code to the Agoric testnet, usually for the purpose of running a contract and modifying it based on the result of a governance vote. To do so, we submit a proposal using cosmos-sdk level API ([swingset.CoreEval](https://community.agoric.com/t/blder-dao-governance-using-arbitrary-code-injection-swingset-coreeval/99)). If the proposal passes its governance vote, its JS code is run with ocaps extracted using the proposal's permit list, which the code can combine to perform privileged tasks.
```

Reply:
The sentence has been changed to your suggestion.